PyInstaller/depend/bindepend.py

```diff
@@ -875,6 +876,36 @@ def get_python_library_path():
     # Applies only to non Windows platforms.
 
     if is_unix:
+
+        # Search if it is an Anaconda distribution
+        if re.findall('/anaconda', sys.executable):
```

Comment:
Please use `compat.is_conda`.

Beside this:
- Using `re` for searching for static strings is much too much overhead.
- Assuming `/` in the path does not work on e.g. Windows.

Reply:
I actually tried ```usiing compat.is_conda``` but it returns a false value, and I use Anaconda.

Reply:
What is the output of `.../anconda/bin/python -c "import sys; print sys.version"`?